README.md

```diff
@@ -314,14 +353,23 @@ $> git clone https://github.com/horovod/horovod.git
 
 #### Steps
 
-The following is an exmaple command for running the PyTorch Horovod benchmark
+The following is an example command for running the PyTorch Horovod benchmark
 on two nodes with four GPUs per node. Note that the TF benchmark would be
 nearly identical and should just require changing the name of the image to
 that of the TF container and updating the path to the TF2 benchmark.
 
+##### SS Containerized Libraries
+
 ```
 $> cd horovod
-$> srun --mpi=pmi2 -n 8 --ntasks-per-node=4 --distribution=*:block --cpu-bind=socket --ntasks-per-socket=1 -c 72 singularity run --nv --bind /projects --bind $TMPDIR --bind $HOME --bind `pwd` /opt/cray/libfabric/1.15.2.0/lib64:/host/lib,/usr:/host/usr /projects/benchmarking/public/sif/pytorch-ngc-hpc-dev-hvd.sif /projects/benchmarking/public/examples/wrapper.sh python ./examples/pytorch/pytorch_synthetic_benchmark.py --batch-size=96 --fp16-allreduce >& pt-syn-n8-ppn4.txt
+$> srun --mpi=pmix_v4 -n 8 --ntasks-per-node=4 --distribution=*:block --cpu-bind=socket --ntasks-per-socket=1 -c 72 singularity run --nv --bind /projects --bind $TMPDIR --bind $HOME --bind `pwd` /path/to/sif/pytorch-ngc-hpc-dev-0050511.sif python ./examples/pytorch/pytorch_synthetic_benchmark.py --batch-size=96 --fp16-allreduce >& pt-syn-n8-ppn4.txt
+```
+
+##### Bind-mount SS Libraries
+
+```
+$> cd horovod
+$> srun --mpi=pmi2 -n 8 --ntasks-per-node=4 --distribution=*:block --cpu-bind=socket --ntasks-per-socket=1 -c 72 singularity run --nv --bind /projects --bind $TMPDIR --bind $HOME --bind `pwd` --bind /opt/cray/libfabric/1.15.2.0/lib64:/host/lib,/usr:/host/usr /projects/benchmarking/public/sif/pytorch-ngc-hpc-dev-hvd.sif /projects/benchmarking/public/examples/wrapper.sh python ./examples/pytorch/pytorch_synthetic_benchmark.py --batch-size=96 --fp16-allreduce >& pt-syn-n8-ppn4.txt
```

Comment:
Is this "pytorch-ngc-hpc-dev-hvd.sif " to show case that different image is being used compare to "pytorch-ngc-hpc-dev-0050511.sif" in the example?